debian/changelog

```diff
@@ -11,8 +11,7 @@ cloud-init (25.1-0ubuntu1~20.04.1) UNRELEASED; urgency=medium
     - d/p/retain-ec2-default-net-update-events.patch
     - d/p/retain-netplan-world-readable.patch
     - d/p/revert-551f560d-cloud-config-after-snap-seeding.patch
-  * Upstream snapshot based on upstream/main at 1a9393ee.
-  * refresh patches:
+  * sort patch hunks: --sort quilt refresh flag
```

Comment:
I don't think we should list the patches here and just mention "all patches" unless we already had any previously sorted patches, because this makes it look like only 3 patches got sorted.